### PR TITLE
Updated information about module documentation artifacts

### PR DESCRIPTION
**Pull request by quintesse**
@gavinking these are the changes I would suggest to the spec about documentation artifacts.

The change described here is because documentation (like resources) is for user-visible purposes not available as a zip file anymore but only as a directory. Before we were generating both, the idea being that the directory was useful for browsing the docs locally while the zip was for speedy downloading/uploading to the Herd.

But since then the CMR has been extended to (un)zip folders automatically when needed so the .zip archives for documentation weren't needed anymore.

### CODE REVIEW

**Comment by quintesse** [comment]
For #189

**Comment by gavinking** [comment]
So we no longer have any sort of packaged documentation, not even a zip?

**Comment by quintesse** [comment]
There's a download link for a zip on the Herd

**Comment by gavinking** [comment]
So it looks to me like there still is a `.doc.zip` archive. Not sure why you deleted that material from the spec...

**Comment by quintesse** [comment]
Well because it's nothing formal anymore, there's nothing that depends on that name being exactly as it is.

File: en/modules/modules.xml [hunk 1]
```diff
@@ -523,31 +506,42 @@
...
             
-            <para>Inside a documentation archive, HTML source is located in the 
-            <literal>module-doc</literal> directory.</para>
+            <para>Inside a documentation folder, HTML source for the API documentation
+            is located in the <literal>api</literal> directory.</para>
+            
+            <para>Inside a documentation folder, any files related to user documentation
+            are located in the <literal>doc</literal> directory.</para>
             
             <para>Thus, the structure of the documentation archive for the module
```
Comment (gavinking) [inline en/modules/modules.xml]: @quintesse is this supposed to say "documentation folder"?

**Comment by gavinking** [comment]
So if it's "nothing formal", I would say the spec doesn't even need to cover this...

**Comment by gavinking** [comment]
The module doc folder was already covered later in the spec.

So I've just completely deleted the two sections about module archives, instead of merging this.

Thanks anyway, @quintesse :)

File: en/modules/modules.xml (hunk 1 above)
Comment (quintesse) [inline en/modules/modules.xml]: Yes, documents, like resources, are stored as a bunch of files in a folder. Only for transfer purposes to/from a remote server like the Herd are they zipped.

**Comment by quintesse** [comment]
As you wish, as long as it's not incorrect anymore at least :)

**Pull request closed without merging**